python/lsst/sims/movingObjects/chebyshevUtils.py

```diff
@@ -9,7 +9,7 @@
 
 # Evaluation routine.
 
-def chebeval(x, p, interval=(-1., 1.), doVelocity=True):
+def chebeval(x, p, interval=(-1., 1.), doVelocity=True, mask=False):
```

Comment:
There are a lot of loops in this method.  It seems like we should be able to accomplish the same thing with numpy arrays.  I do not think that is in-scope for this ticket, but maybe take it as an issue in the future (with all of that free development time I'm sure we'll have...)